src/shellbar.scss

```diff
@@ -229,5 +286,37 @@ $block: #{$fd-namespace}-shellbar;
         display: inline-block;
       }
     }
+
+    .fd-counter--notification {
+      max-width: 2.5rem;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      position: absolute;
+      background: $fd-content-badge-background;
+      border: $fd-content-badge-background;
+      color: $fd-shellbar-color;
+    }
+
+    .fd-popover__control {
+      cursor: text;
+
+      .fd-input-group__addon {
+        background-color: transparent;
+      }
+
+      .fd-input-group {
+        background-color: transparent;
+
+        &:hover {
+          background-color: $fd-shell-hover-background;
+          border-color: $fd-field-border-color;
+        }
+
+        .fd-input-group__input {
+          border: none;
+          color: $fd-shellbar-color;
+        }
```

Comment:
`fd` should be replaced by `#{$fd-namespace}`
#{$fd-namespace}-counter--notification